src/components/FeatureViews/AdminPortal/UnpaidSessions.js

```diff
@@ -15,19 +16,26 @@ function UnpaidSessions() {
 
     const studentStatus = hooks.useStudent(studentList);
     const courseStatus = hooks.useCourse(courseList);
-    const useUnpaidSessionStatus = hooks.useUnpaidSessions();
+    const unpaidSessionStatus = hooks.useUnpaidSessions();
 
-    if (hooks.isLoading(studentStatus, courseStatus, useUnpaidSessionStatus)){
-        return <Loading/>
+    if(hooks.isSuccessful(unpaidSessionStatus) && UnpaidList.length === 0) {
+        return <Card>
+            <CardContent>
+                <Typography variant="h5">
+                    No unpaid sessions to display!
+                </Typography>
+                <HappyIcon fontSize="large"/>
+            </CardContent>
+        </Card>
     }
 
-    if(UnpaidList.length === 0) {
-        return <>No unpaid sessions to display</>
-    } else {
+    if (hooks.isLoading(studentStatus, courseStatus, unpaidSessionStatus)){
+        return <Loading small loadingText="UNPAID SESSIONS LOADING"/>
+    }
 
     return UnpaidList.map(unpaidStudent=>(
             <UnpaidSessionCard unpaidStudent={unpaidStudent}/>
```

Comment:
Add `key` prop to elements in an array

Reply:
@gglinoga  i missed this in my previous PR review for you too lol